packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-mobile.tsx

```diff
@@ -269,10 +327,10 @@ const Duration = observer(
             return trade_basis;
         };
 
-        const onTabChange = index => {
+        const onTabChange = (index: number) => {
             setDurationTabIdx(index);
             const { value: unit } = duration_units_list[index];
-            setSelectedDuration(unit, duration_values[`${unit}_duration`]);
+            setSelectedDuration(unit, duration_values[`${unit as 't' | 's' | 'm' | 'h' | 'd'}_duration`]);
```

Comment:
`duration_values` keys are `t_duration` ant etc (line 307), so it's not possible to use something like `keyof typeof duration_values`